chargeOclock/Source/Views/ClientTableViewCell.swift

```diff
@@ -41,20 +41,19 @@ extension ClientTableViewCell: UITextFieldDelegate {
 		client.name = text
 		client.save { error in
 			guard error == nil else {
-				let alert = UIAlertController(title: "Save error", message: "Client saving failed, error \(error?.localizedDescription)", preferredStyle: .alert)
+				let alert = UIAlertController(title: "Save error", message: "Client saving failed, with error:\n\(error)", preferredStyle: .alert)
```

Comment:
I left it out as I found that `localizedDescription` doesn't work as I thought. I found out that using just the error the displayed text corresponds to the name of the error case (eg. *internalError*, *serverError(401)*, ...). In contrast, using the `localizedDescription` displayed some unfamiliar, useless description.

But I did some search now and I see (I guess 🤔) the "right" way should be to use `localizedDescription` and also implement the `localizedDescription` computed property in `oClockError`.